config/environments/staging.rb

```diff
@@ -78,8 +78,8 @@
   # Send deprecation notices to registered listeners
   config.active_support.deprecation = :notify
 
-  API_HOST = 'http://api.digitalnz.org'
-  API_KEY = 'gV1woMjzQzsbzakmXXpb'
+  API_HOST = 'http://api.dnz0a.digitalnz.org/' #'http://api.digitalnz.org'
+  API_KEY =  'bES4B3UwR64cP8msCphF' #'gV1woMjzQzsbzakmXXpb'
```

Comment:
Unnecessary spacing detected.<br>Missing space after #.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.